cpp/server/handler.cc

```diff
@@ -1,237 +1,78 @@
-#include "server/handler.h"
-
 #include <algorithm>
 #include <event2/buffer.h>
 #include <event2/http.h>
-#include <event2/keyvalq_struct.h>
 #include <functional>
 #include <gflags/gflags.h>
 #include <glog/logging.h>
-#include <map>
 #include <memory>
+#include <mutex>
+#include <stdint.h>
 #include <stdlib.h>
+#include <string>
 #include <utility>
 #include <vector>
 
-#include "log/cert.h"
-#include "log/cert_checker.h"
-#include "log/cluster_state_controller.h"
 #include "log/frontend.h"
 #include "log/log_lookup.h"
-#include "log/logged_entry.h"
 #include "monitoring/monitoring.h"
 #include "monitoring/latency.h"
-#include "server/json_output.h"
-#include "server/proxy.h"
-#include "util/json_wrapper.h"
-#include "util/thread_pool.h"
-
-namespace libevent = cert_trans::libevent;
-
-using cert_trans::Cert;
-using cert_trans::CertChain;
-using cert_trans::CertChecker;
-using cert_trans::Counter;
-using cert_trans::HttpHandler;
-using cert_trans::JsonOutput;
-using cert_trans::Latency;
-using cert_trans::LoggedEntry;
-using cert_trans::Proxy;
-using cert_trans::ScopedLatency;
+#include "server/handler.h"
```

Comment:
Ah, this is because this file briefly turned into a template (remember?) and then back out to a regular class.
I've moved the class header back up top, though.